lib/arel/extensions/prepare_statement.rb

```diff
@@ -0,0 +1,53 @@
+# rubocop:disable Naming/MethodName
```

Comment:
Could you make two separate files `lib/arel/extensions/dealocate.rb` and `lib/arel/extensions/prepare.rb`?